src/settings.c

```diff
@@ -0,0 +1,4 @@
+#include "settings.h"
+
+float length = 1.50;
```

Comment:
You've already declared these two variables in settings.h, so you can omit the variable type (float).